client/ayon_core/pipeline/delivery.py

```diff
@@ -383,6 +383,13 @@ def get_representations_delivery_template_data(
             continue
 
         template_data = repre_entity["context"]
+        # Bug in 'ayon_api', 'get_representations_hierarchy' did not fully
+        #   convert representation entity. Fixed in 'ayon_api' 1.0.10 .
```

Comment:
```suggestion
        #   convert representation entity. Fixed in 'ayon_api' 1.0.10.
```

Reply:
Feels weird to have dot after version with dots.

Reply:
Haha, I understand - but as far as I know that's how grammar works. 👼